src/styles/annotator/notebook.scss

```diff
@@ -24,4 +27,29 @@
   padding: 0;
   width: 100%;
   height: 100%;
+  border: none;
```

Comment:
Can you help me understand this rule? Was a border showing up? That is, is this rule necessary?

Reply:
There was a 1px grey border showing up. It came from `@include molecules.panel` in the `.notebook-inner`. 

In the `.Notebook__close-buton` if `position: absolute`, it will show the border between the iframe and the parent container, in this way:
![image](https://user-images.githubusercontent.com/8555781/109005087-d8931300-76a9-11eb-8374-5b2aae2c35d0.png)